### ISSUE TITLE
sklearn is giving better performance than autl-sklearn on my own dataset

### ISSUE THREAD

**Issue by tiru1930** [automl/auto-sklearn#929]
I am using sklearn MultinomialNB and auto-sklearn MultinomialNB  as below in my pipe line , and i am using same features but i got sklearn classfier is giving better results than auto-sklearn. 

I am not sure about what went wrong in the configuration 

I have tried in increase the time_left_for_this_task but got same results.  

only thing i can think of i am not sending feature_type in model fit.  May I know how can debug more on this, weather its data problem or configuration problem 

```
model_clf = autosklearn.classification.AutoSklearnClassifier(include_estimators=["multinomial_nb"],time_left_for_this_task=500,n_jobs=-1)
#         model_clf = MultinomialNB()

```

## Environment and installation: ##

Please give details about your installation:

* OS: linux docker
* conda 
* Python version : 3.8
* Auto-sklearn : 0.8

**Comment by tiru1930** [comment]
@mfeurer sorry for the late reply 

Please find below logs

```
FOR USER:  rc@gmail.com
TOTAL COUNT OF TRAINING TRIPS:  285
TOTAL COUNT OF VALIDATION TRIPS:  71
[WARNING] [2020-08-24 15:19:04,778:AutoML(1872583848):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.707144)
[WARNING] [2020-08-24 15:19:04,779:AutoML(4153361530):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.722050)
[WARNING] [2020-08-24 15:19:04,781:AutoML(1872583848):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,796:AutoML(4153361530):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,823:AutoML(3834805130):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.695515)
[WARNING] [2020-08-24 15:19:04,820:AutoML(327741615):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.708965)
[WARNING] [2020-08-24 15:19:04,826:AutoML(3834805130):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,836:AutoML(327741615):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,833:AutoML(953453411):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.676702)
[WARNING] [2020-08-24 15:19:04,849:AutoML(3751350723):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.691785)
[WARNING] [2020-08-24 15:19:04,854:AutoML(953453411):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,852:AutoML(3751350723):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,879:AutoML(774252441):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.693537)
[WARNING] [2020-08-24 15:19:04,900:AutoML(774252441):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,923:AutoML(2365658986):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,918:AutoML(2365658986):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.576605)
[WARNING] [2020-08-24 15:19:04,922:AutoML(662124363):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.661432)
[WARNING] [2020-08-24 15:19:04,932:AutoML(662124363):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,965:AutoML(1791095845):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.618558)
[WARNING] [2020-08-24 15:19:04,968:AutoML(1791095845):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:04,977:AutoML(44556670):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.573099)
[WARNING] [2020-08-24 15:19:05,006:AutoML(44556670):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:19:05,023:AutoML(3312796937):bf104a83ec768af9ef264cc97eae8ead] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.536700)
[WARNING] [2020-08-24 15:19:05,044:AutoML(3312796937):bf104a83ec768af9ef264cc97eae8ead] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:19,068:smac.runhistory.runhistory.RunHistory] Encountered exception Expecting ',' delimiter: line 382 column 43 (char 8205) while reading runhistory from /tmp/autosklearn_tmp_256eac3e-e61d-11ea-8546-0242ac180002/smac3-output/run_1872583848/runhistory.json. Not adding any runs!
(201, 95)
[WARNING] [2020-08-24 15:27:22,794:AutoML(1872583848):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.719265)
[WARNING] [2020-08-24 15:27:22,801:AutoML(1872583848):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,811:AutoML(2365658986):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.709862)
[WARNING] [2020-08-24 15:27:22,818:AutoML(4153361530):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.710750)
[WARNING] [2020-08-24 15:27:22,814:AutoML(2365658986):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,833:AutoML(3834805130):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.712244)
[WARNING] [2020-08-24 15:27:22,835:AutoML(4153361530):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,835:AutoML(3834805130):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,855:AutoML(327741615):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.699496)
[WARNING] [2020-08-24 15:27:22,871:AutoML(3751350723):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.693845)
[WARNING] [2020-08-24 15:27:22,892:AutoML(774252441):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.704762)
[WARNING] [2020-08-24 15:27:22,879:AutoML(327741615):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,895:AutoML(774252441):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,884:AutoML(3751350723):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,903:AutoML(44556670):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.673666)
[WARNING] [2020-08-24 15:27:22,896:AutoML(3312796937):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.690924)
[WARNING] [2020-08-24 15:27:22,908:AutoML(44556670):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,908:AutoML(3312796937):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,945:AutoML(662124363):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.664423)
[WARNING] [2020-08-24 15:27:22,977:AutoML(1791095845):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.630291)
[WARNING] [2020-08-24 15:27:22,979:AutoML(1791095845):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:22,976:AutoML(662124363):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in each process.
[WARNING] [2020-08-24 15:27:23,014:AutoML(953453411):5302dfe11dc76b22a2745171d947d0c1] Time limit for a single run is higher than total time limit. Capping the limit for a single run to the total time given to SMAC (499.623647)
[WARNING] [2020-08-24 15:27:23,033:AutoML(953453411):5302dfe11dc76b22a2745171d947d0c1] Capping the per_run_time_limit to 249.0 to have time for a least 2 models in

```

I will try with dev branch of auto-ml

Reason I was using MultinomialNB this is my base line for existing work-flow. so I need compare these results.

**Comment by mfeurer** [comment]
Hey @tiru1930  unfortunately this is only the head of the log file. Could you please check in the directory given via the argument `tmp_folder` for the logfile and upload the whole file to github (and not paste it)?

**Comment by stupidoge** [comment]
@mfeurer Hello, Matthias. I met the same problem when running multi-processing in auto-sklearn. Although there was a bug regarding one unknow variable (I fixed it later), I still met problems such as 
`[WARNING] [2023-11-26 15:03:18,309:Client-AutoML(1):c2174ec3-8c29-11ee-8582-cc96e50794f5] Capping the per_run_time_limit to 21599.0 to have time for a least 2 models in each process.` 
and 
`[WARNING] [2023-11-26 15:03:19,746:Client-EnsembleBuilder] No runs were available to build an ensemble from`

I have uploaded the log file into [google drive](https://drive.google.com/file/d/1UatH6mZHaMZI4BfByAETbDJIBRQ6q9yw/view?usp=sharing).
